test/utils/shippable/sanity-requirements.txt

```diff
@@ -2,3 +2,5 @@ tox
 pyyaml
 jinja2
 setuptools
+botocore
+boto3
```

Comment:
Changes to this file aren't needed since the dependencies are in the `test/utils/tox` requirements files.